app/src/main/res/layout/activity_reminder.xml

```diff
@@ -113,15 +113,15 @@
         android:text="@string/do_not_send_me_reminders"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintStart_toEndOf="parent"
-        app:layout_constraintBottom_toTopOf="@+id/reminder_done_button"/>
+        app:layout_constraintBottom_toTopOf="@id/reminder_done_button"/>
```

Comment:
What happens if we change `+id` to `id`?

Reply:
because + shoul be ONLY when you create a View. better to not have the same names in the project also. Here is a pointer to View

Reply:
```
        _
       /(|
      (  :
     __\  \  _____
   (____)  `|
  (____)|   |
   (____).__|
    (___)__.|_____
```